qiskit/circuit/qpy_serialization.py

```diff
@@ -1033,32 +1033,32 @@ def _write_pauli_evolution_gate(file_obj, evolution_gate):
     num_operators = len(operator_list)
     pauli_data_buf = io.BytesIO()
     for operator in operator_list:
-        element_buf = io.BytesIO()
-        buf = io.BytesIO()
-        pauli_list = operator.to_list(array=True)
-        np.save(buf, pauli_list)
-        data = buf.getvalue()
-        element_metadata = struct.pack(SPARSE_PAULI_OP_LIST_ELEM_PACK, len(data))
-        element_buf.write(element_metadata)
-        element_buf.write(data)
-        pauli_data_buf.write(element_buf.getvalue())
+        with io.BytesIO() as element_buf:
+            with io.BytesIO() as buf:
```

Comment:
No need to change this, but for the future you can avoid an extra scope by doing
```python
with io.BytesIO() as element_buf, io.BytesIO() as buf:
```
the scope order is guaranteed to be left-to-right. (Assuming it's not a line-length constraint!)

Reply:
Ah cool, I didn't realize that